plenty_api/api.py

```diff
@@ -1308,6 +1310,17 @@ def plenty_api_get_shipping_packages_for_order(self, order_id: int,
         return utils.summarize_shipment_packages(
             response=package_responses, mode=mode)
 
+    def plenty_api_get_amazon_product_types(self):
+        """
+        Get a list of all available Amazon product types.
+        https://developers.plentymarkets.com/en-gb/plentymarkets-rest-api/index.html#/Pim/get_rest_pim_amazon_product_types
+        Return:
+                        [JSON(Dict) / DataFrame] <= self.data_format
+        """
+        domain = 'pim' # pim is the REST Route not the domain
```

Comment:
You don't use the variable. And I don't think you should as it is a very short string. (Actually the string is shorter than the variable name :laughing:)

Reply:
You are absolutely right! 
It was more about a place for the comments, since naming is rather confusing, I would say.


```python
        domain = 'pim' # pim is the REST Route not the domain
        path = '/amazon-product-types' # path in this case is the endpoint
```

And staying in line with other API GET call function styles. 